assets/menu.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 36.08 13.67"><title>Asset 1</title><text transform="translate(0 10.28)" style="font-size:12px;fill:#253b80;font-family:OpenSans-Semibold, Open Sans;font-weight:700">MENU</text></svg>
```

Comment:
This SVG uses a text element - should we just use text in the markup to simplify things?